routes/web.php

```diff
@@ -44,22 +44,15 @@
 /*-----------------------*
  * Device Controller       *
  *-----------------------*/
-Route::resource('device', 'DeviceController');
 Route::put('device/{id}/restore', 'DeviceController@restore')->name('device.restore');
-// Ajax calls
-Route::get('device/{site_id}/locations', 'DeviceController@locations');
-Route::get('device/{id}/details', 'DeviceController@details');
+Route::resource('device', 'DeviceController');
```

Comment:
NBD, but why place the resource last?

Reply:
Not really needed for what we have but https://laravel.com/docs/5.5/controllers#restful-supplementing-resource-controllers says the routes should be like this.

Reply:
Gotcha, thanks!